crates/bevy_winit/src/lib.rs

```diff
@@ -25,6 +26,7 @@ use bevy_ecs::prelude::*;
 use bevy_window::{exit_on_all_closed, Window, WindowCreated};
 pub use system::create_windows;
 use system::{changed_windows, despawn_windows};
+pub use winit::event_loop::EventLoopProxy;
```

Comment:
Out of curiosity: This means `EventLoopProxy` gets re-exported, right? Was this kept as some sort of backwards compatibility since the type alias was also being publicly exposed?

Reply:
Mostly just because it's good policy to reexport public types, so downstream users don't have to take a dependency on winit if they need the inner type.